doc/man3/OPENSSL_VERSION_NUMBER.pod

```diff
@@ -86,17 +86,19 @@ if available or "ENGINESDIR: N/A" otherwise.
 
 For an unknown B<t>, the text "not available" is returned.
 
-=head1 RETURN VALUE
+=head1 RETURN VALUES
 
-The version number.
+OpenSSL_version_num() returns the version number.
+
+OpenSSL_version() returns version strings in different format.
```

Comment:
I'd rather say "requested [immutable?] version string". I mean it has lesser to do with format, but different information...